javaslang/src/main/java/javaslang/collection/LinkedHashMultimap.java

```diff
@@ -192,12 +192,12 @@ public ContainerType getContainerType() {
 
     @Override
     public <K2, V2> LinkedHashMultimap<K2, V2> map(BiFunction<? super K, ? super V, Tuple2<K2, V2>> mapper) {
-        return null; // TODO ...
+        return Multimaps.map(this, emptyInstance(), mapper);
```

Comment:
would it be possible to force push these, otherwise we're reviewing the same line multiple times (even though they were fixed in a later commit).